static/src/js/components/CollectionDetails/CollectionDetailsCloudAccessInstance.scss

```diff
@@ -0,0 +1,80 @@
+.collection-details-cloud-access-direct-distribution-information {
+  &__info {
+      > dt {
```

Comment:
This whole block should be using 2 spaces per tab. Check your settings in vs code and make sure you have it set to the proper value. This might be the same issue affecting the block I called out in `static/src/js/components/CollectionDetails/CollectionDetailsCloudAccessInstance.js`

Reply:
Yea that was it. It was on 4 spaces per tab thank you for that catch